README.md

```diff
@@ -16,20 +16,17 @@ MixedReality-WebRTC is a collection of libraries to help mixed reality app devel
 
 MixedReality-WebRTC is part of the collection of repositories developed and maintained by the [Mixed Reality Sharing team](https://github.com/orgs/microsoft/teams/mixed-reality-sharing).

```

Comment:
Remove "active development" badge at top of file.

Reply:
Done.